app/controllers/cart_controller.rb

```diff
@@ -7,6 +7,7 @@ def add_item
   end
 
   def show
+    render file: "/public/404" if current_admin?
```

Comment:
Solves admin being able to see cart

Reply:
This is awesome! I think this is a great solution!